components/phpunit_bridge.rst

```diff
@@ -358,8 +380,19 @@ If you have installed the bridge through Composer, you can run it by calling e.g
 
     $ vendor/bin/simple-phpunit
 
+.. tip::
+
+    Set the ``SYMFONY_PHPUNIT_VERSION`` env var to e.g. ``5.5`` to change the
+    base version of PHPUnit to ``5.5`` instead of the default ``5.3``.
+
+.. tip::
+
+    Set the ``SYMFONY_PHPUNIT_REMOVE`` env var to ``symfony/yaml`` if you need
+    ``prophecy`` but not ``symfony/yaml``.
```

Comment:
What does "if you need" mean? Am I not able to use Prophecy if I don't set this env var to `symfony/yaml`?